lib/Page.js

```diff
@@ -128,24 +128,21 @@ Page.prototype.cleanUpDependency = function (file) {
  * Pre-render an external dynamic dependency to the same path as the current page
  * @param file
```

Comment:
Remove this wrong jsdoc or update it :)

Reply:
"same path as the current page" has been wrong after #3 (MarkBind/markbind#70).